src/main/javascript/src/components/authentication/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  Button, Form, Icon, Input,
+} from 'antd';
+import { FormComponentProps } from 'antd/es/form';
+
+import './login-form.scss';
+import { postLogin } from '../../../lib/api/login';
+import { AuthenticationState } from '../types';
+
+const BLOCK = 'login-form';
```

Comment:
I tried to use the BEM convention in the project
```suggestion
const BLOCK = 'authentication_login-form';
```